sinker/otel/components.go

```diff
@@ -53,18 +56,20 @@ func StartOtelComponents(ctx context.Context, bridgeService *bridgeservice.Sinke
 	transformCtx := context.WithValue(otelContext, "component", "transformprocessor")
 	log.Info("start to create component", zap.Any("component", transformCtx.Value("component")))
 	transformCfg := transformFactory.CreateDefaultConfig().(*transformprocessor.Config)
-	transformCfg.OTTLConfig.Metrics.Statements = []string{
-		`set(attributes["agent-name"], resource.attributes["agent_name"])`,
-		`set(attributes["agent-tags"], resource.attributes["agent_tags"])`,
-		`set(attributes["orb-tags"], resource.attributes["orb_tags"])`,
-		`set(attributes["agent-groups"], resource.attributes["agent_groups"])`,
-		`set(attributes["agent-ownerID"], resource.attributes["agent_ownerID"])`,
-		`set(attributes["policy-id"], resource.attributes["policy_id"])`,
-		`set(attributes["policy-name"], resource.attributes["policy_name"])`,
-		`set(attributes["sink-id"], resource.attributes["sink_id"])`,
-		`set(attributes["format"], "otlp")`,
-	}
-	transformSet := component.ProcessorCreateSettings{
+	// transformCfg.MetricStatements = []transformprocessor.ContextStatements{
+	// 	{Statements: []string{
+	// 		`set(attributes["agent-name"], resource.attributes["agent_name"])`,
+	// 		`set(attributes["agent-tags"], resource.attributes["agent_tags"])`,
+	// 		`set(attributes["orb-tags"], resource.attributes["orb_tags"])`,
+	// 		`set(attributes["agent-groups"], resource.attributes["agent_groups"])`,
+	// 		`set(attributes["agent-ownerID"], resource.attributes["agent_ownerID"])`,
+	// 		`set(attributes["policy-id"], resource.attributes["policy_id"])`,
+	// 		`set(attributes["policy-name"], resource.attributes["policy_name"])`,
+	// 		`set(attributes["sink-id"], resource.attributes["sink_id"])`,
+	// 		`set(attributes["format"], "otlp")`,
+	// 	}},
+	// }
+	transformSet := processor.CreateSettings{
```

Comment:
why comment? if it is not going to be used anymore, clean that up.

Reply:
oh sorry, I will fix it

Reply:
done